site/gatsby-site/plugins/netlify-plugin-process-notifications/index.js

```diff
@@ -44,6 +40,8 @@ const processNotifications = async () => {
   return processResult;
 };
 
+processNotifications();
```

Comment:
I think this `processNotifications()` call shouldn't be here. We already call it inside the `onSuccess` function